server/views/pages/prisonerList/partials/searchTable.njk

```diff
@@ -1,78 +1,39 @@
 {# TODO: Logic for the problemRetrievingData #}
 {% if not ciagPrisonerList.problemRetrievingData %}
-  {{ govukTable({
-    attributes: {
-      'data-module': 'moj-sortable-table'
-    },
-    head: [
-      {
-        text: "Prisoner",
-        attributes: {
-          "aria-sort": "ascending"
-        }
-      },
-      {
-        text: "Location",
-        attributes: {
-          "aria-sort": "none"
-        }
-      },
-      {
-        text: "Release date",
-        attributes: {
-          "aria-sort": "none"
-        }
-      },
-      {
-        text: "Entered prison on",
-        attributes: {
-          "aria-sort": "none"
-        }
-      },
-      {
-        text: "Status",
-        attributes: {
-          "aria-sort": "none"
-        }
-      }
-    ],
-    rows: [
-      [
-        {
-          html: "<a href='/plan/G9981UK/view/overview' rel='noopener noreferrer' class='govuk-link govuk-link--no-visited-state'>Avary, Austin</a><br><span class='govuk-hint govuk-!-margin-bottom-0'>G9981UK</span>"
-        },
-        {
-          text: "A-1-032"
-        },
-        {
-          text: "3 February 2020"
-        },
-        {
-          text: "13 December 2021"
-        },
-        {
-          text: ""
-        }
-      ],
-      [
-        {
-          html: "<a href='/plan/G5045UF/view/overview' rel='noopener noreferrer' class='govuk-link govuk-link--no-visited-state'>Aishard, Ogonanthomasin</a><br><span class='govuk-hint govuk-!-margin-bottom-0'>G5045UF</span>"
-        },
-        {
-          text: "B-1-002"
-        },
-        {
-          text: "N/A"
-        },
-        {
-          text: "12 July 2023"
-        },
-        {
-          html: "<strong class='govuk-tag govuk-tag--red'>Needs plan</strong>"
-        }
-      ]  
-    ]
-  }) }}
+  <table class="govuk-table" data-module="moj-sortable-table">
```

Comment:
Using standard HTML here opposed to Nunjucks, it would have been a bit of work to do in Nunjucks macros, but may revisit this in the future.